google/cloud/logging/_gax.py

```diff
@@ -18,18 +18,17 @@
 
 from google.gax import CallOptions
 from google.gax import INITIAL_PAGE
-from google.gax.errors import GaxError
 from google.logging.type.log_severity_pb2 import LogSeverity
 from google.logging.v2.logging_config_pb2 import LogSink
 from google.logging.v2.logging_metrics_pb2 import LogMetric
 from google.logging.v2.log_entry_pb2 import LogEntry
 from google.protobuf.json_format import Parse
 from grpc import StatusCode
+from grpc._channel import _Rendezvous
```

Comment:
> Are we really going to commit

Was hoping to resolve discussion in #2156. Happy to add an alias of our own.

I think the real issue though is 0.13.0 of GAX. Like I think they meant to be catching those `_Rendezvous` and giving us `GaxError`s to catch.